docs/source/features.rst

```diff
@@ -101,6 +107,12 @@ articles:
 OMEMO in Multi-user chats (MUC)
 -------------------------------
 
+.. note::
+    For end-to-end encryption via OMEMO, you'll need to load `libsignal-protocol.js
+    <https://github.com/signalapp/libsignal-protocol-javascript>`_ separately in
+    your page. Take a look at the section on :ref:`libsignal <dependency-libsignal>` and the
+    :ref:`security considerations around OMEMO <feature-omemo>`.
+
```

Comment:
You already have the note in the same page above, perhaps a bit excessive to have it twice?

Reply:
I did that because it's a section that can be linked to with an anchor (making it easy to miss the note in the section above). I'm happy to remove it if you think that it's overkill.

Reply:
Yeah, I can see your point but I still think it's a bit much. I'd prefer it if you'd remove it, thanks.

Reply:
![image](https://user-images.githubusercontent.com/4253898/172581569-7530bf90-7090-4e66-a3b4-b134c86dbfe6.png)